src/sql/src/plan/with_options.rs

```diff
@@ -236,6 +236,24 @@ impl ImpliedValue for bool {
     }
 }
 
+#[derive(Default, Debug, Copy, Clone)]
+pub struct OnlyImpliedTrue(pub bool);
+
+impl TryFromValue<Value> for OnlyImpliedTrue {
+    fn try_from_value(_: Value) -> Result<Self, PlanError> {
+        sql_bail!("cannot specify value")
+    }
+    fn name() -> String {
+        "only_implied_true".to_string()
+    }
+}
+
+impl ImpliedValue for OnlyImpliedTrue {
+    fn implied_value() -> Result<Self, PlanError> {
+        Ok(OnlyImpliedTrue(true))
+    }
+}
```

Comment:
I would rather not add this. This is complexity that we will likely forget to remove. It also is likely that someone will use this in a non-unsafe option in the future, at which point it becomes product debt, because we will have some special boolean options that do not support the consistent `OPTION = {true|false}` syntax. The fact that all boolean options support that syntax is a feature!

Reply:
sounds good!